v2-migration/src/main/java/software/amazon/awssdk/v2migration/ChangeSdkType.java

```diff
@@ -133,20 +137,6 @@ private static boolean isWildcard(String fullName) {
                    fullName.matches(V1_SERVICE_WILD_CARD_CLASS_PATTERN);
         }
 
-        private static boolean isV1Class(JavaType.FullyQualified fullyQualified) {
```

Comment:
This method was rewritten and moved to SdkTypeUtils. It's now `isSupportedV1Class`